crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -940,6 +951,22 @@ enum MoveVisibility {
 	FRIEND
 }
 
+type Mutation {
+	"""
+	Execute a transaction, committing its effects on chain.
+	
+	`txBytes` is a `TransactionData` struct that has been BCS-encoded
+	and then Base64-encoded.
+	`signatures` are a list of `flag || signature || pubkey` bytes,
+	Base64-encoded.
+	
+	Waits until the transaction has been finalised on chain to return
+	its transaction digest.  If the transaction could not be
+	finalised, returns the errors that prevented it, instead.
```

Comment:
Sorry 😬 my Britishness is leaking through these comments -- can you Americanize them before landing?

```suggestion
	Waits until the transaction has been finalized on chain to return
	its transaction digest.  If the transaction could not be
	finalized, returns the errors that prevented it, instead.
```